src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -10,17 +10,13 @@
 		- search
 		- space
 		- yoo
-		
 	MCB:
 		- app
 		- bangumi
-		- passport
```

Comment:
Mixed Content: The page at 'https://passport.bilibili.com/site/site.html' was loaded over HTTPS, but requested an insecure script 'http://data.bilibili.com/a/access.js?_=1483357511280'. This request has been blocked; the content must be served over HTTPS.